src/App.jsx

```diff
@@ -14,33 +17,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_KEYS = {
+  alphabet: 'alphabet',
+  length: 'length',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortBy, setSortBy] = useState(''); // lenght, alphabet
+  const [reverse, setReverse] = useState(1);
+  const [resetActive, setResetActive] = useState(false);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const resetButtons = () => {
+    document.querySelectorAll('.button').forEach(button => {
+      button.classList.add('is-light');
+    });
+  };
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  const sortedGoods = goodsFromServer.toSorted((goodA, goodB) => {
+    if (sortBy === 'alphabet') {
+      return goodA.localeCompare(goodB);
+    }
+
+    if (sortBy === 'length') {
+      return goodA.length - goodB.length;
+    }
+
+    return 0;
+  });
+
+  const reverseGoods = reverse < 0 ? sortedGoods.toReversed() : sortedGoods;
```

Comment:
The `toReversed` method is not a standard JavaScript array method. Consider using `Array.prototype.reverse()` instead, which is the standard method for reversing arrays.